features/step_definitions/monitoring.rb

```diff
@@ -2,12 +2,12 @@
 
   if infrastructure("cluster").infra_topology=="SingleReplica"
 
-    step %Q/there are 1 "daemonset" replicas in the "openshift-monitoring" project/
-    step %Q/there are 1 "statefulset" replicas in the "openshift-monitoring" project/
-    step %Q/there are 1 "deployment" replicas in the "openshift-monitoring" project/
-    step %Q/there are 1 "daemonset" replicas in the "openshift-monitoring" project/
-    step %Q/there are 1 "statefulset" replicas in the "openshift-user-workload-monitoring " project/
-    step %Q/there are 1 "deployment" replicas in the "openshift-user-workload-monitoring" project/
+    step %Q/there is 1 "daemonset" replicas in the "openshift-monitoring" project/
```

Comment:
@lihongyan1 to be grammatically correct, please replace `replicas` with `replica`